sechub-systemtest/src/main/java/com/mercedesbenz/sechub/systemtest/runtime/launch/SystemTestRuntimeProductLauncher.java

```diff
@@ -211,9 +211,9 @@ public void waitUntilSecHubAdminAvailable(SystemTestRuntimeContext context) {
 
         for (int i = 0; !adminAccountAvailable && i < MAX_AMOUNT_OF_ADMIN_AVAILABLE_CHECKS; i++) {
             try {
-                sechubClient.fetchSecHubStatus();
+                sechubClient.atOtherApi().adminListStatusInformation();
```

Comment:
My intention was to fully replicate the structure of the doc so that users can match the doc with the client exactly

"Other" is a section inside the doc

If we change it I would like to change it there as well